cmd/object-api-utils.go

```diff
@@ -239,6 +239,8 @@ func getCompleteMultipartMD5(parts []CompletePart) string {
 func cleanMetadata(metadata map[string]string) map[string]string {
 	// Remove STANDARD StorageClass
 	metadata = removeStandardStorageClass(metadata)
+	// Remove Tags
+	delete(metadata, xhttp.AmzObjectTagging)
 	// Clean meta etag keys 'md5Sum', 'etag', "expires".
 	return cleanMetadataKeys(metadata, "md5Sum", "etag", "expires")
```

Comment:
Interestingly enough only one function is necessary here 

```
return cleanMetadataKeys(metadata, "md5Sum", "etag", "expires", xhttp.AmzObjectTagging, xhttp.AmzStorageClass)
```

Reply:
:+1:

Reply:
`removeStandardStorageClass` is needed because the filtering is based on the value instead of key